# HG changeset patch
# User Thorsten Meyer <thorsten.meyier@gmx.de>
# Date 1235985737 -3600
#      Mon Mar 02 10:22:17 2009 +0100
# Node ID ca032de5fbf295013457db5722b8e5788e2645b6
# Parent  fd83cc87f55e9657a98934c78690eefc828002b7
Remove references to deprecated function create_set.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2009-03-02  Thorsten Meyer  <thorsten.meyier@gmx.de>
+
+	* set/complement.m, set/union.m: Remove reference to deprecated
+	function create_set.
+
 2009-03-01  Ben Abbott <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Do not render axis labels twice when
 	their handles are visible, but do render objects with hidden
 	handles.
 
 2009-03-01  Ben Abbott <bpabbott@mac.com>
 
diff --git a/scripts/set/complement.m b/scripts/set/complement.m
--- a/scripts/set/complement.m
+++ b/scripts/set/complement.m
@@ -23,17 +23,17 @@
 ## example,
 ##
 ## @example
 ## @group
 ## complement ([ 1, 2, 3 ], [ 2, 3, 5 ])
 ##      @result{} 5
 ## @end group
 ## @end example
-## @seealso{create_set, union, intersect}
+## @seealso{union, intersect, unique}
 ## @end deftypefn
 
 ## Author: jwe
 
 function y = complement (a, b)
 
   if (nargin != 2)
     print_usage ();
diff --git a/scripts/set/union.m b/scripts/set/union.m
--- a/scripts/set/union.m
+++ b/scripts/set/union.m
@@ -43,17 +43,17 @@
 ## @end group
 ## @end example
 ##
 ## @deftypefnx {Function File} {[@var{c}, @var{ia}, @var{ib}] =} union (@var{a}, @var{b})
 ##
 ## Return index vectors @var{ia} and @var{ib} such that @code{a == c(ia)} and
 ## @code{b == c(ib)}.
 ## 
-## @seealso{create_set, intersect, complement}
+## @seealso{intersect, complement, unique}
 ## @end deftypefn
 
 ## Author: jwe
 
 function [y, ia, ib] = union (a, b, varargin)
 
   if (nargin < 2 || nargin > 3)
     print_usage ();
